server/storage/memory_test.go

```diff
@@ -90,3 +91,135 @@ func TestGetChecksumNotFound(t *testing.T) {
 	require.Error(t, err)
 	require.IsType(t, ErrNotFound{}, err)
 }
+
+func TestMemoryGetChanges(t *testing.T) {
+	s := NewMemStorage()
+
+	// non-int changeID
+	c, err := s.GetChanges("foo", 10, "")
+	require.Error(t, err)
+	require.Len(t, c, 0)
+
+	// add some records
+	require.NoError(t, s.UpdateMany("alpine", []MetaUpdate{
+		{
+			Role:    data.CanonicalTimestampRole,
+			Version: 1,
+			Data:    []byte{},
+		},
+		{
+			Role:    data.CanonicalTimestampRole,
+			Version: 2,
+			Data:    []byte{},
+		},
+		{
+			Role:    data.CanonicalTimestampRole,
+			Version: 3,
+			Data:    []byte{},
+		},
+		{
+			Role:    data.CanonicalTimestampRole,
+			Version: 4,
+			Data:    []byte{},
+		},
+	}))
+	require.NoError(t, s.UpdateMany("busybox", []MetaUpdate{
```

Comment:
Was wondering if we can insert a `Delete` here or after the busybox updates?  So that we can test that the delete events get returned, etc.  Also, I think this is the same as the sql tests?  (creating the same changes, testing the change feed, etc.) - would it make sense to move the implementation into `tuf_store_test.go` (or we can rename that file or something) as `testGetChanges`, and then invoke it with either a memory store or a sql store?

Similarly, can we add a `Delete` of a GUN that doesn't exist to the end to ensure that no change object is added in that case?